hw/ip/csrng/data/csrng_testplan.hjson

```diff
@@ -92,9 +92,7 @@
             - which_hw_inst_exc (0 to NHwApps-1), NHwApps possible hw instance exceptions
             - which_sp2v (0 to Sp2VWidth-1), SP2V_HIGH_LOGIC width
             - which_fatal_err (0 to 25), 26 possible fatal errors
-            - which_fifo_write_err (0 to 15), 16 different fifos
-            - which_fifo_read_err (0 to 15), 16 different fifos
-            - which_fifo_state_err (0 to 15), 16 different fifos
+            - which_fifo (0 to 15), 16 different fifos
```

Comment:
In EDN you have a cover point to highlight which kind of error occured.  Is that needed here?

Reply:
Yes, it's covered by which_fifo_err below to specify which type of fifo error.